setup.py

```diff
@@ -16,35 +16,8 @@
 
 from distutils.core import Command
 
-from admin.release import make_rpm_version
-
-class cmd_generate_spec(Command):
-    description = "Generate python-flocker.spec with current version."
-    user_options = []
-    boolean_options = []
-    def initialize_options(self):
-        pass
-    def finalize_options(self):
-        pass
-    def run(self):
-        with open('python-flocker.spec.in', 'r') as source:
-            spec = source.read()
-
-        flocker_version = versioneer.get_version()
-        version, release = make_rpm_version(flocker_version)
```

Comment:
We should probably file a ticket about making this use ``characteristic`` instead of ``namedtuple``, now that it isn't being used in `setup.py`.